src/asm3/dbupdate.py

```diff
@@ -6312,3 +6312,8 @@ def update_34902(dbo: Database) -> None:
         dbo.ddl_add_table_column("IsRetired", dbo.type_integer, True)
     ])
     dbo.execute_dbupdate( dbo.ddl_add_table("lkmediaflags", fields) )
+
+def update_34903(dbo: Database) -> None:
+    # Add extra fields to facilitate invoice tracking to animalcost table
+    add_column(dbo, "animalcost", "OwnerID", dbo.type_integer)
+    add_column(dbo, "animalcost", "InvoiceNumber", dbo.type_shorttext)
```

Comment:
Need to add indexes for both these columns. Also, as well as writing a database update function, you need to add these columns to the definition of the animalcost table above in the sql_structure function (and the indexes again).

This is because there are two paths, one for setting up a brand new database, and one for updating existing databases.